WifiART/src/userinterface/ArtMvcView.java

```diff
@@ -373,6 +423,122 @@ private void setupZPanel(JPanel panel) {
 
     }
 
+    private void setupUsagePanel(JPanel panel) {
```

Comment:
why set the sub-panel layouts as FlowLayout?

Reply:
I use FlowLayouts as a way of left-justifying the components in the panel. Each row has a FlowLayout so that the components are pushed to the left side of the frame.